Documentation

beninato8 opened the pull request:
Splitting up #201 into 3. This PR is just the documentation directory.

I also added a new directory with a bit of documentation. I'm not sure if there is another location with more documentation, but I wanted somewhere to put my notes as I've been working, and hopefully it could be useful for others who want to contribute.

pvpoke reviewed (commented):
Thanks for these notes! These seem more appropriate for the wiki (see the relevant wiki page here and if there's any way you'd like to incorporate them).

https://github.com/pvpoke/pvpoke/wiki/Gamemaster

beninato8 commented:
Yeah sorry about that, I found the wiki after you linked me to it and forgot to mention that in this one. I'll looking seeing if I can add them there.

beninato8 commented:
I've removed my docs directory, and linked to the wiki in the README with this PR. I added the information to the wiki on my own version here: https://github.com/beninato8-forks/pvpoke/wiki. 

I believe the [steps here](https://stackoverflow.com/a/56810747) should allow you to merge the changes I made to my local wiki repo into the wiki repo for the main repository:

```
$ git clone git@github.com:pvpoke/pvpoke.wiki.git
$ cd pvpoke.wiki
$ git pull https://github.com/beninato8-forks/pvpoke.wiki.git master
$ git push origin master
```

If you have any feedback on how I laid out the changes, let me know!
